gitlab-base.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: gitlab-base
+  version: 16.4.0
+  epoch: 0
+  description: GitLab Base is the base image, which other GitLab containers can build on top of.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - bash
+      - gomplate
+      - gitlab-logger
+      - busybox
+      - xtail
+      - curl
+      - ca-certificates-bundle
+      - procps
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/local/bin/ ${{targets.destdir}}/scripts
+      install -m755 -D "./exec-env" ${{targets.destdir}}/scripts
+      install -m755 -D "./set-config" ${{targets.destdir}}/scripts
+      install -m755 -D "./entrypoint.sh" ${{targets.destdir}}/scripts
+  - uses: strip
+
+update:
```

Comment:
Do we want to create a new project in release-monitoring ? Or follow other gitlab version updates ?

Reply:
I'll reuse the one from cng as it looks similar.